### ISSUE TITLE
Performance issue on orbitdb while saving objects

### ISSUE THREAD

**Issue by rakeshbhatt10** [orbitdb/orbitdb#1029]
Hi,

We are trying to push data to orbitdb. 

We are using docstore to save our data. We are using ipfsjs 

IPFS configuration
```
{
        repo:"repo",
        start: true,
        repo:"repo",
        start: true,
        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
        //@ts-ignore
        preload: {
          enabled: true,
        },
        EXPERIME
        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
        //@ts-ignore
        preload: {
          enabled: true,
        },
        EXPERIMENTAL: {
          ipnsPubsub: true,
        },
        relay: {
          enabled: true,
          hop: {
            enabled: true,
          },
        },
        config: {
          Addresses: {
            Swarm:
              typeof window === 'undefined'
                ? [
                    // When running outside of the browser we can support TCP connections
                    `/ip4/0.0.0.0/tcp/${process.env.PORT || 4005}/`,
                    `/ip4/0.0.0.0/tcp/${process.env.WS_PORT || 4020}/ws`
                  ]
                : [
                    // When running on a browser we need to use a signaling server, below is the Steer Protocol signaling server
                    '/dns4/guarded-sands-38279.herokuapp.com/tcp/443/wss/p2p-webrtc-star/',
                  ],
          },
          Bootstrap: [
          ],
        
          Discovery: {
            // MDNS: {
            //   Enabled: true,
            //   Interval: 10
            // },
            webRTCStar: {
              Enabled: true,
            },
          },`

```

All data that is being pushed to OrbitDb is through a scheduled tasks which runs on fixed intervals

On first iteration we can see on given address that objects are getting inserted easily. 

In second iteration if we try to store objects inside same address it is slow

Next iteration it is unable to put objects in OrbitDB

Do let us know in case of any other information is required

Regards,

**Comment by inspiraluna** [comment]
I do not understand if the storage to the local machine is slow in your case or the replication to another node is slow? Also are you using nodejs as peers or browser nodes? Or both mixed?

**Comment by haadcode** [comment]
This should be fixed in 1.0.0. Note the [new package name](https://www.npmjs.com/package/@orbitdb/core).

If the issue persists, feel free to re-open the issue.
